### PR TITLE
Features/attestation

### PR DESCRIPTION
**Pull request by sarabiamark7098**

### CODE REVIEW

File: resources/views/pdf/attestation.blade.php [hunk 1]
```diff
@@ -96,7 +96,7 @@
             </div>
             <div style="margin-left: 6%; margin-right: 6%; font-size: 16px;">
                 <div style="margin-top: 50px; text-align: justify; text-indent: 40px;">
-                    This is to certify that Mr./Mrs. {{ $client['first_name'] }}@if(!empty($client['middle_name'])) {{ $client['middle_name'][0] }}.@endif {{ $client['last_name'] }}@if(!empty($client['ext_name'])) {{ $client['ext_name'] }}@endif, {{ $client['age'] }} years old
```
Comment (jpgulayan460103) [inline resources/views/pdf/attestation.blade.php:99]: Try `nullsafe` operator instead `if else` master

Comment (jpgulayan460103) [inline resources/views/pdf/attestation.blade.php:99]: try `null coalescing` operator master instead na `if else`, pero verify sa php7.3 if working

https://www.tutorialspoint.com/php/php_null_coalescing_operator.htm

File: resources/js/Components/App.vue [hunk 2]
```diff
@@ -103,6 +103,43 @@ export default {
...
+      },
+      /*{
+        to: "/assistance-types",
+        text: "Assistance Types",
+      },
+      {
+        to: "/import-served",
+        text: "Import Served",
+
       },*/
```
Comment (jpgulayan460103) [inline resources/js/Components/App.vue:143]: Remove nani master, pwede raman ni i retrieve later

File: resources/views/auth/login.blade.php [hunk 3]
```diff
@@ -67,7 +67,7 @@ class="form-control @error('password') is-invalid @enderror" name="password" req
                     </div>
                 </form>
                 <div>
-                    <a href="Register">Register</a>
```
Comment (jpgulayan460103) [inline resources/views/auth/login.blade.php:70]: kani sad master, retrieve lng later

File: resources/views/pdf/coe_batch.blade.php [hunk 4]
```diff
@@ -186,7 +186,7 @@
                     <td style="width:15%"> {{ $aics_beneficiary['payroll_client']['sequence'] }}</td>
                     <td></td>
                     <td style="width:25%">
-                        {{ date('m-d-Y', strtotime($aics_beneficiary['payroll_client']['created_at'])) }}</td>
```
Comment (jpgulayan460103) [inline resources/views/pdf/coe_batch.blade.php:189]: kani sad master

**Review by jpgulayan460103 — COMMENTED** [review]
Remove lng mga commented lines master, pwede raman ma retrieve later

File: resources/views/pdf/attestation.blade.php [hunk 5]
```diff
@@ -96,7 +96,7 @@
...
+                    This is to certify that <b>Mr./Mrs. {{ $client['first_name'] }} {{ $client['middle_name']?$client['middle_name'][0].". ":"" }}{{ $client['last_name'] }}{{ $client['ext_name']?" ".$client['ext_name']:"" }}</b>, {{ $client['age'] }} years old
```
Comment (jpgulayan460103) [inline resources/views/pdf/attestation.blade.php:99]: ay tama rani nga `ternary` operator. dli diay mugana ang `null coalescing` dri ky naay index 0 haha

File: resources/views/pdf/coe_batch.blade.php (hunk 4 above)
Comment (jpgulayan460103) [inline resources/views/pdf/coe_batch.blade.php:189]: apila nani ug remove master kung dli na gamiton

File: resources/views/pdf/coe_batch.blade.php [hunk 6]
```diff
@@ -335,7 +335,7 @@
                 <td style="width:15%; text-align:left;">QN: {{ $aics_beneficiary['payroll_client']['sequence'] }}</td>
                 <td></td>
                 <td style="width:25%">
-                    {{ date('m-d-Y', strtotime($aics_beneficiary['payroll_client']['created_at'])) }}</td>
```
Comment (jpgulayan460103) [inline resources/views/pdf/coe_batch.blade.php:338]: kani pud

File: resources/views/pdf/gis.blade.php [hunk 7]
```diff
@@ -269,9 +269,9 @@
         style=" font-weight:bold; text-align:center;top:50pt;right:20pt; width:10px;height:10px;padding:10px; border: dotted 1px; border-radius:50%; ">
         {{ $aics_beneficiary['payroll_client']['sequence'] }}</div>
     <div class="data-textbox" style="top:86pt;right:20pt;">
-        {{ \Carbon\Carbon::parse($aics_beneficiary['payroll_client']['created_at'])->timeZone('Asia/Manila')->format('m-d-Y') }}</div>
+        <!-- {{ \Carbon\Carbon::parse($aics_beneficiary['payroll_client']['created_at'])->timeZone('Asia/Manila')->format('m-d-Y') }}</div> -->
     <div class="data-textbox" style="top:103pt;right:20pt;">
-        {{ \Carbon\Carbon::parse($aics_beneficiary['payroll_client']['created_at'])->timeZone('Asia/Manila')->format('h:i a')}}</div>
```
Comment (jpgulayan460103) [inline resources/views/pdf/gis.blade.php:274]: kani pud

File: resources/views/pdf/gis_many.blade.php [hunk 8]
```diff
@@ -124,7 +124,7 @@
             {{ $aics_beneficiary['payroll_client']['sequence'] }}
         </div>
         <div class="data-textbox" style="top:80pt;right:75pt;">
-            {{ \Carbon\Carbon::parse($aics_beneficiary['payroll_client']['created_at'])->timeZone('Asia/Manila')->format('m-d-Y') }}
```
Comment (jpgulayan460103) [inline resources/views/pdf/gis_many.blade.php:127]: kani pud master

File: resources/views/pdf/attestation.blade.php (hunk 5 above)
Comment (sarabiamark7098) [reply resources/views/pdf/attestation.blade.php:99]: dli mugana master

File: resources/views/pdf/attestation.blade.php (hunk 1 above)
Comment (sarabiamark7098) [reply resources/views/pdf/attestation.blade.php:99]: dli mugana ang null coalescing

File: resources/views/pdf/attestation.blade.php (hunk 5 above)
Comment (jpgulayan460103) [reply resources/views/pdf/attestation.blade.php]: gani master hahaha, tungod sa index 0

**Review by jpgulayan460103 — APPROVED** [review]

**Pull request merged**